libs/deal-ticket/src/components/deal-ticket/deal-ticket-error.tsx

```diff
@@ -0,0 +1,52 @@
+import type { ReactNode } from 'react';
+import classNames from 'classnames';
+import { InputError } from '@vegaprotocol/ui-toolkit';
+import { useEffect, useRef, useState } from 'react';
+
+export interface DealTicketErrorMessage {
+  message: ReactNode | string;
+  isDisabled: boolean;
+}
+
+interface Props {
+  errorMessage?: DealTicketErrorMessage;
+  'data-testid'?: string;
+}
+
+export const DealTicketError = ({
+  errorMessage,
+  'data-testid': dataTestId = 'deal-ticket-error-message',
+}: Props) => {
+  const [message, setMessage] = useState<DealTicketErrorMessage['message']>('');
+  const ts = useRef<null | number>(null);
+  useEffect(() => {
```

Comment:
chore (blocking): I think it would be better to leave this extra complexity out for now. Once the design team has given us some animation guidelines we can come back and animate consistently with a suitable library

Reply:
ok, as you wish, I'm gonna remove animation completely for now. I just wanted to avoid really ugly effect of "jumping" sections when some error messages appear and hide. These few additional lines (of useState and useEffects) seemed to me as not too complicated and worked nice as well  :-( . But ok, I'm gonna remove it ofc, especially if we have in plans to do  a consistent solution.